frontend/src/components/about/CitationWidget.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="is-flex is-justify-content-flex-start">
+    <a
+      :href="plumxref()"
+      data-popup="right"
+      data-size="medium"
+      class="plumx-plum-print-popup"
+      data-site="plum"
+      data-hide-when-empty="true"
+    >
+      An atlas of human metabolism
```

Comment:
This text is shown quickly and then replaced by the widgets. Is there any special reason to show this text?

Reply:
That's a good point. This is a fallback text but we don't have any fallback for the other widgets so I think it makes sense to not have any here as well.

Reply:
Fixed in 58d4d4a792372e01047277608059d56147e61912